DESCRIPTION

```diff
@@ -44,15 +45,16 @@ Depends:
 Imports:
     BBmisc (>= 1.11),
     backports,
-    ggplot2,
-    stats,
-    stringi,
     checkmate (>= 1.8.2),
     data.table,
+    ggplot2,
     methods,
     parallelMap (>= 1.3),
+    stats,
+    stringi,
     survival,
-    utils
+    utils,
+    stringr
```

Comment:
pls dont change the normal imports. you have also now added stringr, while we use stringi